<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bfh_addquery</class>
 <widget class="QWidget" name="bfh_addquery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>项目控制-新增记录</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>40</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>新增</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>100</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>清空内容</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>706</x>
     <y>21</y>
     <width>34</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;年龄&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_age">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>21</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_applycr">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>22</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>22</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;申报金额(万)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>69</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;申报产品&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="add_applyproduct">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>67</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>惠农担-粮食贷</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>惠农担-生猪贷</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>惠农担-茶叶贷</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>惠农担-油茶贷</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>惠农担-特色贷</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>惠农担-农机贴息贷</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;循环项目&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="add_loopitem">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>67</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>是</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>否</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="add_id">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>21</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_name">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>20</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>21</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;身份证号码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>20</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;客户名称&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;联系方式&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_cpnumber">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>70</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;授信开始时间&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="add_sxstarttime">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>120</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;授信结束时间&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="add_sxendtime">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>120</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd/</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="add_process">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>118</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>请选择</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>预受理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>调查走访</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>评审流程</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>合同签订</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>批复变更</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>银行信息变更</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>核保放款</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>确认放款</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>循环项目银行信息变更</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>循环项目核保放款</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>循环项目确认放款</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>项目展期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>委托代偿</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>被动代偿</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>项目还款</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>712</x>
     <y>120</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;当前流程&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_comments">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>711</x>
     <y>162</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;备注&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>add_name</tabstop>
  <tabstop>add_id</tabstop>
  <tabstop>add_age</tabstop>
  <tabstop>add_applycr</tabstop>
  <tabstop>add_applyproduct</tabstop>
  <tabstop>add_loopitem</tabstop>
  <tabstop>add_cpnumber</tabstop>
  <tabstop>add_sxstarttime</tabstop>
  <tabstop>add_sxendtime</tabstop>
  <tabstop>add_process</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
